xml/x11_banners.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+X11 vendor fields (part of the X11 success response) are matched 
```

Comment:
Yeah that's the same field. I'll point to the protocol doc https://www.x.org/archive/X11R7.5/doc/x11proto/proto.pdf and add some additional explanation.